src/components/Renderer/types.ts

```diff
@@ -28,6 +28,8 @@ export interface UiSchema
 	'ui:title'?: string;
 	'ui:description'?: string;
 	'ui:value'?: Value;
+	'ui:explicit'?: boolean;
+	items?: UiSchema;
```

Comment:
These are used and expected to be typed like this at runtime so I added them here to better represent the true type of UiSchema